clang/lib/Sema/SemaCodeComplete.cpp

```diff
@@ -1837,6 +1837,10 @@ static void AddTypeSpecifierResults(const LangOptions &LangOpts,
       Builder.AddChunk(CodeCompletionString::CK_RightParen);
       Results.AddResult(Result(Builder.TakeString()));
     }
+
+    if (LangOpts.CPlusPlus20) {
```

Comment:
That can also be enabled by `LangOpts.Char8`

Reply:
resolve.